docs/dev/stream/state/schema_evolution.zh.md

```diff
@@ -25,17 +25,16 @@ under the License.
 * ToC
 {:toc}
 
-Apache Flink streaming applications are typically designed to run indefinitely or for long periods of time.
-As with all long-running services, the applications need to be updated to adapt to changing requirements.
-This goes the same for data schemas that the applications work against; they evolve along with the application.
+Apache Flink 流应用通常被设计为永远或者长时间运行。
+与所有长期运行的服务一样，需要更新应用程序以适应不断变化的需求。
+对于应用程序处理的数据结构来说，这是相同的；它们也随应用程序一起升级。
 
-This page provides an overview of how you can evolve your state type's data schema. 
-The current restrictions varies across different types and state structures (`ValueState`, `ListState`, etc.).
+此页面概述了如何升级状态类型的数据结构。
```

Comment:
Maybe we should not translate `data schema`[1]

[1] https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications